tests/pytorch/test_pytorch_metric_value_convertion_utils.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from unittest import mock
+
+import mlflow
+from mlflow import tracking
+from mlflow.tracking.fluent import start_run
+from mlflow.exceptions import MlflowException, INVALID_PARAMETER_VALUE, ErrorCode
+from mlflow.tracking.metric_value_conversion_utils import *
```

Comment:
Can we avoid using wild car import?

Reply:
Of course! will also update other 2 tests that use this.